README.md

```diff
@@ -9,10 +9,15 @@ The Slack Orb for CircleCI is now written in Go! This means the orb is now more
 1. The orb now downloads the Slack Orb CLI binary to `.circleci/orbs/circleci/slack/$PLATFORM/$ARCH` in the working directory.
    1. This can be checksum verified
    2. This is cacheable if desired.
-2. You can no longer use evaluated sub-shell commands (e.g. `$(date +%s)`).
+2. The `custom` parameter used for custom inline templates has been renamed to `template_inline`.
+3. The `template` parameter is now reserved only for built-in templates and will no longer support dynamic templates
+4. Dynamic templates must now be defined with the `template_var` parameter.
+5. You can no longer use evaluated sub-shell commands (e.g. `$(date +%s)`).
    1. Instead, you can pre-populate environment variables in the `$BASH_ENV` file. [See more in our wiki]()
+      1. **Known issue:** variables do not expand in Alpine
```

Comment:
Variables do expand since we manually source BASH_ENV [here](https://github.com/CircleCI-Public/slack-orb-go/blob/main/packages/cli/config/config.go#L97). Sub-shells are the problem. Since we don't run `eval echo` on BASH_ENV, they don't get evaluated.

Reply:
Sorry yes, my mistake on wording.